html/rendering/the-css-user-agent-style-sheet-and-presentational-hints/ua-stylesheet-text-decoration-skip.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+
+</script>
+</head>
+<body>
+<script>
+setup({explicit_done : true});
```

Comment:
I don't think you need this; adding tests should be fine up to and including the firing of the load event.

Reply:
Calling the test function inline now.